style.css

```diff
@@ -0,0 +1,93 @@
+
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@600&display=swap');
+
+.container {
+  display: flex;
+  flex-direction: column;
+  min-height: 100vh;
+}
+
+/* styles related to .toolbar and navigation */
```

Comment:
- [ ] Please let's keep our code clean by remove comments because this makes the code noisy. 👍